meta-resin-common/recipes-support/resin-conf-reset/resin-conf-reset/resin-conf-reset

```diff
@@ -9,8 +9,10 @@ populate() {
     # Re-construct directory structure and populate files
     if [ -f $2 ]; then
         mkdir -p "$(dirname $1)" && cp $2 $1
-    else
+    elif [ -d $2 ]; then
         mkdir -p $1 && find $2 \( -type f -o -type l \) -exec cp {} $1 +
+    else
+        mkdir -p $1
```

Comment:
Variables containing filenames should be quoted, as a space is a legal character in a filename. Moreover, when passed as an argument to a program, it should be delimited from options with "--", in case the filename starts with "-".